src/webpubsub/azext_webpubsub/client.py

```diff
@@ -0,0 +1,112 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+
+import asyncio
+import sys
+import threading
+import json
+import websockets
+from .vendored_sdks.azure_messaging_webpubsubservice import (
+    build_authentication_token
+)
+
+
+async def connect(url):
+    async with websockets.connect(url, subprotocols=['json.webpubsub.azure.v1']) as ws:
+        comment = """
+----------Usage-----------
+joingroup <group-name>              : Join the connection to group
+leavegroup <group-name>             : Leave the connection from group
+sendtogroup <group-name> <message>  : Send message to group
+event <event-name> <message>        : Send event to event handler
+--------------------------
+        """
+        print(comment)
+        publisher = Publisher(ws)
+        publisher.daemon = True
+        publisher.start()
+        while True:
+            print(await ws.recv())
```

Comment:
When a user is typing messages and ws received something, it will breaking the input line in terminal display.

Reply:
I see, but it seems not easy to make it excellence. I think it's acceptable when it's used as a debugging tool.

Reply:
I see. Maybe it's not hard. You can make a switch between typing context and display context by enter. When user in typing context, you can hold the received data form ws. And print them after switch to display context.